packages/ia-wayback-search/stories/ia-wayback-search.stories.js

```diff
@@ -38,5 +38,13 @@ storiesOf('<ia-wayback-search>', module)
         waybackPagesArchived="32 trillion pages"
         waybackHost="archive.org"
       ></ia-wayback-search>
+      <ia-wayback-search
+        waybackPagesArchived="32 trillion pages"
```

Comment:
QUESTION: Verifying this number again. Wish there were a way to put this in a global constant that could be imported into all these components.